EitherT/Applicative

```diff
@@ -4,7 +4,7 @@ in  let Either = ./../Either/Type
 
 in  let EitherT = ./Type
 
-in  let EitherA = ./../Either/Applicative
+in  let EitherA = ./../Either/Applicative/Sequential
```

Comment:
Shouldn’t we split `EitherT/Applicative` as well?

Reply:
Ya fair, never thought about it.